spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configserver/src/test/java/org/springframework/cloud/kubernetes/configserver/KubernetesEnvironmentRepositoryTests.java

```diff
@@ -264,25 +264,32 @@ else if (propertySource.getName().equals("secret.stores.stores-dev.default")) {
 	public void testApplicationPropertiesAnSecretsOverride() throws ApiException {
 		CoreV1Api coreApi = mock(CoreV1Api.class);
 		when(coreApi.listNamespacedConfigMap(eq("default"), eq(null), eq(null), eq(null), eq(null), eq(null), eq(null),
-			eq(null), eq(null), eq(null), eq(null))).thenReturn(CONFIGMAP_DEFAULT_LIST);
+				eq(null), eq(null), eq(null), eq(null))).thenReturn(CONFIGMAP_DEFAULT_LIST);
 		when(coreApi.listNamespacedSecret(eq("default"), eq(null), eq(null), eq(null), eq(null), eq(null), eq(null),
-			eq(null), eq(null), eq(null), eq(null))).thenReturn(SECRET_LIST);
+				eq(null), eq(null), eq(null), eq(null))).thenReturn(SECRET_LIST);
 		when(coreApi.listNamespacedConfigMap(eq("dev"), eq(null), eq(null), eq(null), eq(null), eq(null), eq(null),
-			eq(null), eq(null), eq(null), eq(null))).thenReturn(CONFIGMAP_DEV_LIST);
+				eq(null), eq(null), eq(null), eq(null))).thenReturn(CONFIGMAP_DEV_LIST);
 		KubernetesEnvironmentRepository environmentRepository = new KubernetesEnvironmentRepository(coreApi,
-			kubernetesPropertySourceSuppliers, "default");
+				KUBERNETES_PROPERTY_SOURCE_SUPPLIER, "default");
```

Comment:
`KUBERNETES_PROPERTY_SOURCE_SUPPLIER` this was missing, probably a small merge issue. The rest is formatting, nothing else